util/codec/codec.go

```diff
@@ -442,6 +452,28 @@ func SetRawValues(data []byte, values []types.Datum) error {
 	return nil
 }
 
+// decodeVInt decodes the first encoded value as a varint from b and returns varint and its length.
+func decodeVInt(b []byte) (length int, n int64, err error) {
+	if len(b) < 1 {
+		return 0, 0, errors.New("invalid encoded key")
+	}
+	flag := b[0]
+	length++
+	b = b[1:]
```

Comment:
How about call `DecodeVarint` directly?
The length is `len(remainedBytes) - len(b)`.